app/src/main/res/drawable/thumb_code42.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="200dp" android:viewportHeight="240"
+    android:viewportWidth="240" android:width="200dp" xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
It would be nice to have every attribute on a separate line; it gives better readability. See the other vector drawables for comparison.

Reply:
Done